packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1603,15 +1613,26 @@ mixin WidgetInspectorService {
   /// [DiagnosticsNode] object that `diagnosticsNodeId` references providing
   /// information needed for the details subtree view.
   ///
+  /// [subtreeDepth] - the number of levels of the subtree that should be
```

Comment:
Flutter sdk doc comments describing parameters are supposed to be written in a more fluent style. Rewrite this as a sentence describing the parameters of the method instead of as a javadoc style comment with the variable name followed by the value.